src/Ledger/Foreign/LedgerTypes.agda

```diff
@@ -22,7 +22,6 @@ data Empty : Set where
 
 {-# FOREIGN GHC
   data AgdaEmpty deriving (Show, Generic)
-  instance ToExpr AgdaEmpty
```

Comment:
Do you not like `ToExpr` anymore? Should we remove the `tree-diff` dependency?

Reply:
Yeah, I figured it's easier to do as orphan instances on the Haskell side